src/engraving/rendering/score/pagelayout.cpp

```diff
@@ -82,7 +82,7 @@ void PageLayout::getNextPage(LayoutContext& ctx)
     } else {
         state.setPage(dom.pages()[state.pageIdx()]);
         std::vector<System*>& systems = state.page()->systems();
-        state.setPageOldMeasure(systems.empty() ? nullptr : systems.back()->measures().back());
+        state.setPageOldMeasure(systems.empty() || systems.back()->measures().empty() ? nullptr : systems.back()->measures().back());
```

Comment:
Funny enough almost the same code is used in Mu3.7, came in there via https://github.com/Jojo-Schmitz/MuseScore/pull/391. Back then neither I nor the author @jforberg seem to have realized that Mu4 is suffering from the same issue and needs the same fix